spec/features/features_test_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'menu'
+require 'order'
+require 'checkout'
+
+feature 'Placing an order' do
+
+  let(:menu) { Menu.new }
+  let(:order) { Order.new }
+
+  scenario 'Adding dishes to the order' do
+    dish1 = menu.dishes.select {|k, v| k == :'Red Curry'}
```

Comment:
Unused block argument - `v`. If it's necessary, use `_` or `_v` as an argument name to indicate that it won't be used.<br>Space between { and | missing.<br>Space missing inside }.